release_notes.md

```diff
@@ -3,6 +3,7 @@
 <!-- Please add your release notes in the following format:
 - My change description (#PR)
 -->
+- Update Python Worker Version to [4.21.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.21.0)
 - Update Python Worker Version to [4.20.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.20.0)
```

Comment:
Can you remove the release note for 4.20.0?

Reply:
Do we need to remove old versions? I thought this would keep the release history for all versions in this release. I see 4 different powershell versions in this release notes.